de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/inventory/process/M_Inventory_CreateCountLinesFromHU.java

```diff
@@ -117,30 +129,40 @@ private I_M_Inventory getInventory()
 				.iterateAndStream();
 	}
 
-	private Stream<I_M_InventoryLine> createInventoryLines(final I_M_HU hu)
+	private Stream<I_M_InventoryLine> createUpdateInventoryLines(@NonNull final I_M_HU hu, final boolean updateLine)
 	{
 		return Services.get(IHandlingUnitsBL.class)
 				.getStorageFactory()
 				.streamHUProductStorages(hu)
-				.map(huProductStorage -> createInventoryLine(huProductStorage));
+				.map(huProductStorage -> createUpdateInventoryLine(huProductStorage, updateLine));
 	}
 
-	private I_M_InventoryLine createInventoryLine(final IHUProductStorage huProductStorage)
+	private I_M_InventoryLine createUpdateInventoryLine(@NonNull final IHUProductStorage huProductStorage, final boolean updateLine)
```

Comment:
the `updateLine` param is not needed. You could check inventoryLinesByHU directly and decide if create or update.

Reply:
yep; right! thx!